packages/react/menu/src/Menu.tsx

```diff
@@ -79,12 +79,14 @@ const Menu: React.FC<MenuOwnProps> = (props) => {
 
   React.useEffect(() => {
     const handleKeyDown = () => (isUsingKeyboardRef.current = true);
-    const handlePointerMove = () => (isUsingKeyboardRef.current = false);
+    const handlePointer = () => (isUsingKeyboardRef.current = false);
     document.addEventListener('keydown', handleKeyDown, { capture: true });
```

Comment:
by the way, random thought now, but why is capture phase important for all these?

Reply:
Good point, not sure what my reasoning was before 🤔 probably not needed.